wurst/systems/modes/Quests.wurst

```diff
@@ -5,44 +5,73 @@ import LinkedList
 ).joinBy("\n")
 
 let shouts = asList(
-    ENERGY_COLOR.toColorString() + "Authors:|r" + GENERAL_COLOR.toColorString() + " Moop, PL0X_J33BUS, RocketRusty, Quazz, Mephisto.|r",
-    GENERAL_COLOR.toColorString() + "Xel'Naga:|r For creating the original ice trolls",
-    GENERAL_COLOR.toColorString() + "Foosh:|r For being almost as good as me at ice trolls",
-    GENERAL_COLOR.toColorString() + "Clan RPP of USWest",
-    GENERAL_COLOR.toColorString() + "Clan ITTP of Europe.",
-    GENERAL_COLOR.toColorString() + "Clan TwGB of USEast."
+    "Authors:|r".color(ENERGY_COLOR) + " Moop, PL0X_J33BUS, RocketRusty, Quazz, Mephisto, Marsunpaisti, Master-Troll.|r".color(GENERAL_COLOR),
+    "Xel'Naga:|r For creating the original ice trolls".color(GENERAL_COLOR),
+    "Foosh:|r For being almost as good as me at ice trolls".color(GENERAL_COLOR),
+    "Clan RPP of USWest".color(GENERAL_COLOR),
+    "Clan ITTP of Europe.".color(GENERAL_COLOR),
+    "Clan TwGB of USEast".color(GENERAL_COLOR)
 ).joinBy("\n")
 
 let community = asList(
-    "Feel free to join us on discord, ask questions, give your feedback on the map & watch the stream to learn more about the game",
-    "Discord: https://discord.gg/Rh9JdKs",
-    "GitHub: https://github.com/island-troll-tribes/island-troll-tribes",
-    "Current Map Updater: ",
-    "   America server : jmclaus#1870",
-    "   Europe  server :  jaccouille#1782"
+    "Feel free to join us on discord, ask questions, give your feedback on the map & watch the stream to learn more about the game".color(HIGHLIGHT_COLOR),
+    "Discord: ".color(HIGHLIGHT_COLOR) + "https://discord.gg/Rh9JdKs",
+    "GitHub: ".color(HIGHLIGHT_COLOR) + "https://github.com/island-troll-tribes/island-troll-tribes",
+    "Current Map Updater: ".color(HIGHLIGHT_COLOR),
+    "   America server : ".color(HIGHLIGHT_COLOR) + "jmclaus#1870",
+    "   Europe   server : ".color(HIGHLIGHT_COLOR) + "jaccouille#1782"
 ).joinBy("\n")
 
+let credits = asList(
+    "We give our thanks to the following people/community for their help on map making".color(HIGHLIGHT_COLOR),
+    "Wurstscript :".color(HIGHLIGHT_COLOR) + " https://wurstlang.org/".color(ENERGY_COLOR),
+    "HiveWorkShop forum & discord:".color(HIGHLIGHT_COLOR)  + " https://www.hiveworkshop.com/".color(ENERGY_COLOR),
+    "Assets credits:".color(HIGHLIGHT_COLOR),
+    "General Frank " .color(HIGHLIGHT_COLOR) + "for Spirit of water/wind/darkness and gem of knowledge",
+    "Sin'dorei300 "  .color(HIGHLIGHT_COLOR) + "for iron/steel armor, skin coat and sniff icon",
+    "Matarael "      .color(HIGHLIGHT_COLOR) + "for iron/steel/anabolic boots and glove model",
+    "Blood raven "   .color(HIGHLIGHT_COLOR) + "for bone coat",
+    "Usedwell "      .color(HIGHLIGHT_COLOR) + "for hydra coat",
+    "Tranquil "      .color(HIGHLIGHT_COLOR) + "for bear trap",
+    "olofmoleman "   .color(HIGHLIGHT_COLOR) + "for disco duck model",
+    "RightField "    .color(HIGHLIGHT_COLOR) + "for cooked meat and snake mode",
```

Comment:
I'm assuming this is a typo and meant to say model

Reply:
Indeed